pkg/trivyoperator/config.go

```diff
@@ -261,6 +262,15 @@ func (c ConfigData) GetScanJobPodTemplateLabels() (labels.Set, error) {
 	return scanJobPodTemplateLabelsMap, nil
 }
 
+func (c ConfigData) GetScanJobPodPriorityClassName() (string, error) {
```

Comment:
it will be great to add a simple test for this method to protect it

Reply:
> @SamarSidharth I'm planning to cut a release next week , if you could complete the PR by then , it will be included

Sure :)

Reply:
Regarding the test part since the value is coming from the CM data field which is 

> could you also update the [helm docs](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/README.md) ?

Updated